packages/blocks/src/page-block/edgeless/components/connector/connector-handler.ts

```diff
@@ -0,0 +1,126 @@
+import { WithDisposable } from '@blocksuite/lit';
+import type { ConnectorElement } from '@blocksuite/phasor';
+import { DisposableGroup } from '@blocksuite/store';
+import { css, html, LitElement } from 'lit';
+import { customElement, property, query } from 'lit/decorators.js';
+import { styleMap } from 'lit/directives/style-map.js';
+
+import type { EdgelessPageBlockComponent } from '../../edgeless-page-block.js';
+
+@customElement('edgeless-connector-handler')
+export class EdgelessConnectorHandler extends WithDisposable(LitElement) {
```

Comment:
Should be *handle* (as noun) rather than *handler*.

Reply:
ok, I thought handler was noun :)